StepDefinition/LanguageStepDefinitions.cs

```diff
@@ -0,0 +1,138 @@
+
+using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using QAMars.Pages;
+using QAMars.Utilities;
+using System;
+using System.Linq;
+using System.Text;
+using TechTalk.SpecFlow;
+using System.Reflection.Emit;
+using System.Threading.Tasks;
+using System.Collections.Generic;
+using System.Security.Cryptography.X509Certificates;
+using OpenQA.Selenium.DevTools.V128.Page;
+
+namespace ConsoleApp2.QAMars.StepDefinition
+{
+    [Binding]
+    public class LanguageStepDefinitions: CommonDriver
+    {
+        [Given(@"I logged into Mars portal successfully")]
+        public void GivenILoggedIntoMarsPortalSuccessfully()
+        {
+            
+            //login page object initiallization and definition
+            LoginPage loginPageObj = new LoginPage(driver);
+            loginPageObj.LoginActions(driver);
+        }
+        [When(@"I add a language ""([^""]*)"" and language level ""([^""]*)""")]
+        public void WhenIAddALanguageAndLanguageLevel(string language, string level)
+        {
+            //Add language page object initialization and definition
+            LanguageTest LanguagePageObj = new LanguageTest(driver);
+            LanguagePageObj.ClearData(driver);
+            LanguagePageObj.AddLanguage(driver,language, level);
+
+        }
+        //Verify if language record is created successfully
+
+        [Then(@"the ""([^""]*)"" record should be created successfully")]
+        public void ThenTheRecordShouldBeCreatedSuccessfully(string language)
+        {
+            LanguageTest LanguagePageObj = new LanguageTest(driver);
```

Comment:
you shouldnt create a new object every time, instead create an instance variable and then make use of constructors 

e.g 
LanguageTest languagePageObj;

public void LanguageStepDefinitions(){
languagePageObj = new LanguageTest();
}
also please use correct coding standards 

Please make use of constructors, please see 
 https://www.tutorialspoint.com/csharp/csharp_classes.htm

https://www.knowledgehut.com/tutorials/csharp/csharp-constructors

https://www.dofactory.com/csharp-coding-standards

Reply:
Appropriate constructors are added. I have removed the instance of object creation

Reply:
how have you made use of constructors?